api/models/vpp/l3/keys.go

```diff
@@ -100,3 +100,8 @@ func ParseProxyARPInterfaceKey(key string) (iface string, isProxyARPInterfaceKey
 	}
 	return "", false
 }
+
+// RoutePrefix returns longest-common prefix of keys representing route that is written to given vrf table.
+func RoutePrefix(vrf string) string {
```

Comment:
The `vrf` parameter should be `uint32` just like the `VrfId` filed for `Route`: https://github.com/ligato/vpp-agent/blob/f94992578ccc2289b5c123f83e98600699016c52/api/models/vpp/l3/route.pb.go#L50
This will eliminate the need to convert it from every caller.

Reply:
fixed in https://github.com/ligato/vpp-agent/pull/1249/commits/c56066ed17628a796a1a065c609c30a16f01c1e8